### ISSUE TITLE
Sending setwd("[wd]") every time code is sent to R.app

### ISSUE THREAD

**Issue by hathawayj** [hafen/atom-r-exec#1]
Is there a way to stop that from sending every time?

**Comment by pimentel** [comment]
+1 this is a really big issue

**Comment by pimentel** [comment]
FYI, I made a trivial change to fix this:

https://github.com/pimentel/atom-r-exec

I'm not sure why it does this by default? Perhaps the devels can shed some light?

Thanks,

Harold

**Comment by hafen** [comment]
I somehow missed this way back when it was posted.  Sorry @hathawayj.  The only reason it does this by default is that I used a Textmate R plugin back in the day that had this same behavior and I got used to it - to the idea of always wanting to be in the working directory of my project and therefore conservatively sending it with every chunk.  I agree that this probably isn't the most widely desired default behavior.  @pimentel, your changes look good.  If you want to PR, feel free to.  Thanks.
